go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ssvlabs/ssv-spec
 
-go 1.19
+go 1.20
```

Comment:
Why 1.20 and not 1.22?

Reply:
Well, I think I can answer myself: because we are still using 1.20 in the implementation

Reply:
Yes